Source/ACE.Server/Managers/PropertyManager.cs

```diff
@@ -448,7 +448,8 @@ public static void LoadDefaultProperties()
                 ("pk_server", false),
                 ("salvage_handle_overages", false),   // in retail, if 2 salvage bags were combined beyond 100 structure, the overages would be lost
                 ("show_dot_messages", false),        // if enabled, shows combat messages for DoT damage ticks. defaults to disabled, as in retail
-                ("use_wield_requirements", true)    // disable this to bypass wield requirements. mostly for dev debugging
+                ("use_wield_requirements", true),    // disable this to bypass wield requirements. mostly for dev debugging
+                ("world_starts_up_as_closed", false) // enable this to start world as a closed world.
```

Comment:
open_world?
world_closed?

these config vars should be as short as possible

Reply:
changed to world_closed